.github/workflows/verify_consumer_pacts.yml

```diff
@@ -0,0 +1,240 @@
+name: Verify consumer pacts
+# The purpose of this workflow is to verify ANY consumer contract(s) dependent on CBAS provider using Pact framework.
+#
+# The workflow meets the criteria of Pact Broker *Platinum* as described in https://docs.pact.io/pact_nirvana/step_6.
+# The can-i-deploy job has been added to this workflow to gate the merge of PRs into main branch.
+#
+# This workflow is triggered when
+#
+# 1. Consumer makes a change that results in a new pact published to Pact Broker (will verify ONLY the changed pact and publish the verification results back to the broker)
```

Comment:
does the Pact Broker use the `workflow_dispatch` trigger to do this?

Reply:
An excellent question that I bet @ichengchang can answer.

Reply:
@mspector Yes Pact Broker use `workflow_dispatch` to launch this workflow.